bricks/template/__brick__/{{project_name.snakeCase()}}/lib/main.dart

```diff
@@ -41,6 +41,8 @@ class MyApp extends StatelessWidget {
         primarySwatch: Colors.blue,
         brightness: Brightness.light,
         fontFamily: Assets.fonts.neuzeit,
+        colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepPurple),
+        useMaterial3: true,
```

Comment:
I'm not sure if we should use material3 as default - maybe you would want a feature flag to enable it or not when init the project. Besides, this sounds like out-of-scope for the changes you are doing, so I recommend to separate this config in a different PR. Yes - it is small, but it has a big impact.

Reply:
@sleepylee That makes sense. I think we can create a new feature ticket for material3 support with some updates from Theming by @manh-t 🙆‍♂️

Reply:
@sleepylee Remove in https://github.com/nimblehq/flutter-templates/pull/201/commits/108474daa897512fed11601b87d062861aa7d832. Material3 support will be handled in the new task #236

Reply:
@luongvo I will create the ticket and work on that 🙆‍♂️

Reply:
@manh-t let me assign you to this task https://github.com/nimblehq/flutter-templates/issues/236 🙆‍♂️